Lib/pydoc.py

```diff
@@ -1669,9 +1669,11 @@ def render_doc(thing, title='Python Library Documentation: %s', forceload=0,
     return title % desc + '\n\n' + renderer.document(object, name)
 
 def doc(thing, title='Python Library Documentation: %s', forceload=0,
-        output=None):
+        output=None, follow_wrapped=True):
     """Display text documentation, given an object or a path to an object."""
     try:
+        if not follow_wrapped:
```

Comment:
Should be just `follow_wrapped`, right?

Also can we move it outside the `try`, since it should not raise any exception?

Reply:
@chrahunt I use `if follow_wrapped:`